src/uu/wc/src/wc.rs

```diff
@@ -621,63 +621,52 @@ fn compute_number_width(inputs: &[Input], settings: &Settings) -> usize {
     max(minimum_width, total.to_string().len())
 }
 
+/// shorthand for `show!(USimpleError::new(...))`, works just like `format!`.
+macro_rules! record_error {
```

Comment:
Not convinced of the usefulness of this over just using the code. The verboseness of that code should actually incentivize doing the error handling with a custom error type :)

Reply:
Ahhh, yeah, best to avoid making the `USimpleError` at all if we can...  Was that your major objection, or do would you not be okay with,
```rust
macro_rules! record_error {
    ($fmt:literal $(, $arg:expr)*) => {
        eprintln!($fmt $(, $arg)*);
        uucore::errors::set_exit_code(1);
    }
}
```
?

Reply:
I think we should use the code without the custom macro here. However, we can talk in another issue or PR about adding this to uucore instead. At the same time, we can then see whether we can give `show!` a better name. Does that sound good?

Also I made a mistake yesterday, it should be `show_error!` (again a pretty confusing name) not `eprintln!`, because we want to add the name of the util at the start of the line.